uavcan/equipment/air_data/1027.RawAirData.uavcan

```diff
@@ -0,0 +1,41 @@
+#
+# Raw Air Data.
+#
+
+# Note: unused vars should be assigned NaN
```

Comment:
Merits to either approach, 2 messages definetly fits better with the cheap sensors we are all using at the moment, a combined message makes more sense on nicer/higher end equipment in the future. But given that the first use case will probably be cheap sensors that don't include both parts, and that there are already static pressure/temp messages it might make more sense to keep them split up...